types/yup/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for yup 0.26
+// Type definitions for yup 0.27
```

Comment:
Yup 0.27 has not been released (https://github.com/jquense/yup/releases)
I'm unsure about the versioning of DefinitelyTyped projects, but I don't think you need to bump anything unless the feature you added was specifically for 0.27

Reply:
Its a mistake, I was thinking that this number was the version of the current file.

Reply:
fixed :)